packages/lisk-p2p/src/peer.ts

```diff
@@ -30,46 +31,25 @@ export interface IPeerConfig {
 }
 
 export class Peer {
-	private _height: number;
-	private readonly _id: string;
-	private readonly _inboundSocket: any;
```

Comment:
aren't these private variable non-modifiable after constructor?
I think it was ok to be private

Reply:
If we don't make it public then we won't be able to read it from outside without a getter, also, if I change it to private then it will throw lint error `'id' is declared but its value is never read.`

Reply:
yes, originally it had private _id with getter id, so i think that was okay =0